### PR TITLE
Phi Fei Fo Fum

### PR DESCRIPTION
**Pull request by mattansb**
This PR is for writing the Phi Fei (V, W, T) paper, potentially for https://github.com/orgs/easystats/teams/core-team/discussions/5

### CODE REVIEW

**Comment by mattansb** [comment]
Paper is currently a first draft, I invite any one willing to participate to lend a hand in editing / writing / formatting (don't mind switching to qmd instead or rmd, or even MSWord tbh...) @easystats/core-team

**Comment by IndrajeetPatil** [comment]
Team, please include `[skip ci]` in your commit messages and avoid unnecessary energy wastage through CI/CD, which is not relevant for this PR, at least as long as it remains laser-focused on the paper.

**Comment by rempsyc** [comment]
I started working on the paper a bit and made some changes. I had `[skip ci]` in my working memory (and memory pad) right up until I committed the message 🙈 so cancelled all corresponding GHAs after they started. For now, I've fixed the blank פ issue by using unicode. I don't know if having it as a math-mode symbol has a distinct advantage for us here.

**Comment by mattansb** [comment]
Thanks Remi (and ChatGPT)! Please add yourself to the paper (:

**Comment by rempsyc** [comment]
Ok! @mattansb what do you think of this new abstract/intro/conclusion?

**Comment by mattansb** [comment]
Looks good to me! Thanks Remi!

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:61]:
```suggestion
The $\chi^2$ test of independence between two categorical variables examines if the frequency distribution of one of the variables is dependent on the other. That is, are the two variables correlated such that, for example, members of group 1 on variable X are more likely to be members of group A on variable Y, rather than evenly spread across Y variable groups A and B. 
```

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:90]:
```suggestion
For a 2-by-2 contingency table analysis, like the one used above, the $\phi$ (*phi*) coefficient is a correlation-like measure of effect size indicating the strength of association between the two binary variables. One way to compute this effect size is to re-code the binary variables as dummy (0, 1) variables, and computing the Pearson correlation between them:
```

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:118]: What properties?

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:126]:
```suggestion
$V$ is 1 when the columns are completely dependent on the rows, or the row are completely dependent on the columns. That is, when the two variables are perfectly correlated. 
```

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:134]: We should give a counter example of V and Tschuprow tables where V is 1 but the other is not.

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:144]: Definitely show that Matthew's is just Pearson/phi.

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:182]:
```suggestion
We present here a new effect size, `r "\U05E4"` (Fei, pronounced /fej/ or "fay"), which normalizes goodness-of-fit $\chi^2$ by the proper $\chi^2_\text{max}$ for non-uniform and/or multinomial variables.
```

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:253]: We should add a short section to each part above demonstrating not only the tables that produce 0 and 1 but also some "small", "medium", and "large" correlations to help readers get a sense of what different values of these effect sizes look like in data.

File: papers/Phi Fei Fo Fum (WIP)/paper.bib
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.bib:21]:
```suggestion

@article{degeest2010impact,
  title={The impact of research synthesis methods on industrial--organizational psychology: The road from pessimism to optimism about cumulative knowledge},
  author={DeGeest, David S and Schmidt, Frank L},
  journal={Research Synthesis Methods},
  volume={1},
  number={3-4},
  pages={185--197},
  year={2010},
  publisher={Wiley Online Library}
}

@article{wiernik2020unbiased,
  title={Obtaining unbiased results in meta-analysis: The importance of correcting for statistical artifacts},
  author={Wiernik, Brenton M and Dahlke, Jeffrey A},
  journal={Advances in Methods and Practices in Psychological Science},
  volume={3},
  number={1},
  pages={94--123},
  year={2020},
  publisher={Sage Publications Sage CA: Los Angeles, CA}
}
```

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:14]:
```suggestion
- affiliation: 4
  name: Brenton M. Wiernik
  orcid: to add
- affiliation: 5
  name: Daniel Lüdecke 
  orcid: to add

```

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [inline papers/Phi Fei Fo Fum (WIP)/paper.Rmd:22]:
```suggestion
- index: 4
  name: Independent researcher
- index: 5
  name: to add

```

**Review by bwiernik — CHANGES_REQUESTED** [review]
A few revisions and comments on sections we should add for clarity. Looking good!

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (mattansb) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:59]: McNemar's test tests for a difference in the distribution of two measures of a categorical variable.
So if we have fathers' and sons' ADHD diagnosis, we can generate the following table:

| | Father: 1 | 0 | Marginal |
|--|--|--|--|
|Son: 1| 10 | 12 | 22 |
|0|  5| 40 | 45|
| Marginal | 15|52 | |

And asking is {22, 45} different than {15, 52}.

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (mattansb) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:136]: Not sure I would say that, given Tschuprow's *t*...

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (mattansb) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:263]: Not sure what you mean..?

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (mattansb) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:65]: Something like this?
- Odds ratios and risk ratios are only good for 2x2 table
    - RR is also not symmetrical (columns and rows are not exchangeable)
- Pearson's r is appropriate for two qualitative variables (cf. biserial correlation) for which (co)variance is meaningful.
- Cohen's d is applicable for a qualitative outcome (and is not symmetrical) for which a mean and variance are meaningful.

What else?

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (bwiernik) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:281]: in addition to the real-data example, we should also demonstrate, eg, what different "benchmark" small, medium, and large correlation values (e.g., .1, .3, .5, .7, .9) look like in a contingency table

**Comment by bwiernik** [comment]
I can finish out revisions on this tomorrow morning FL time

**Comment by strengejacke** [comment]
@mattansb I'll read this paper after lunch and add my comments/edits

**Comment by mattansb** [comment]
Thanks @strengejacke !

It looks like we won't make it in time for the special issue (? I am also grading papers and finals ☹️), but I think it would still be nice to have this out as a online(-pre)-print (on arxiv?). Does anyone know a good / working template for that?

**Comment by strengejacke** [comment]
I think @DominiqueMakowski is asking for some extra time, since he first announced the 23rd Feb. Is that realistic for you?

**Comment by strengejacke** [comment]
I don't think there's much left. My comments are rather "minor" and mostly requires just an additional sentence or similar. Nothing substantial.

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (mattansb) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:71]: I dropped this.

**Comment by mattansb** [comment]
Did you leave any comments?

I only see comments / suggestions by @bwiernik.

**Review by strengejacke — CHANGES_REQUESTED** [review]

**Comment by strengejacke** [comment]
haha, they were pending, I didn't submit them

**Comment by bwiernik** [comment]
I made a bunch of preprint templates for posting 

https://osf.io/hsv6a/#!

Maybe let's send to Collabora if we can't get into the special issue ?

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (mattansb) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:58]: I think @rempsyc added this?

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (mattansb) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:126]: I've added this in the text. Perhaps @bwiernik (lover of CIs) can take a look...?

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (mattansb) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:255]: I've changed to "will be 0 when the observed distribution matches the expected one **(under the null hypothesis)** perfectly" - these is no good or bad, only statistics (:

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (mattansb) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:256]: > and will be 1 when the sample contains *only* one class of observations---the one with the smallest expected probability (under the null hypothesis). That is, `r "\U05E4"` only when we observe only the least expected class.

How's that?

**Comment by mattansb** [comment]
Okay, I think that's basically it?

Um... Where am I submitting to?

**Comment by strengejacke** [comment]
Nice! Let's wait for @DominiqueMakowski feedback on a possibly extended deadline and then submit to the special issue.

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (strengejacke) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:280]: test comment...

Comment (strengejacke) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:280]: test comment2

File: papers/Phi Fei Fo Fum (WIP)/paper.Rmd
Comment (rempsyc) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:58]: I don't think it's appropriate to define the New Statistics here since we already give a reference to it [@cumming2014new; _The New Statistics: Why and How_]. It should be enough no? If you'd like you can move that reference to the next sentence. And maybe it's not an actual movement it's not that clear so we could remove that word if we want.

Comment (strengejacke) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:58]: > If you'd like you can move that reference to the next sentence

Yeah, maybe that's clearer

Comment (bwiernik) [reply papers/Phi Fei Fo Fum (WIP)/paper.Rmd:58]: Put "new statistics" in quotes as it's a sort of brand name by Cumming and co to refer to estimation + interval approach

**Comment by strengejacke** [comment]
Since deadline-issue seems to be relaxed a bit, we could probably do a final reading the next days (and maybe some revisions, if we think we should do so) and then submit next week or so?

Just a question to the paper title: "Phi, Fei, Fo, Fum: Effect Sizes for Chi-squared Tests" - why fo and fum?

**Comment by strengejacke** [comment]
Regarding submission, see here:
https://www.mdpi.com/journal/mathematics/special_issues/Advances_in_Statistical_Computing

You just need to make sure to mention me including full institutional address for the APC.

**Comment by strengejacke** [comment]
I updated the template like we did for the other paper. However, there's a knit-issue (Unicode) for the Fei character:

> ! LaTeX Error: Unicode character פ (U+05E4)
>               not set up for use with LaTeX.
>
> Error: LaTeX failed to compile paper.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See paper.log for more info.

Any suggestions? @mattansb @rempsyc

**Comment by strengejacke** [comment]
@rempsyc Can you please check if the yaml-header that I revised is correct?

**Comment by mattansb** [comment]
@rempsyc and fix the Unicode issue like you did before? 🙏

**Comment by rempsyc** [comment]
> You just need to make sure to mention me including full institutional address for the APC.

Daniel is it true your institution will only reimburse if you are first/senior author? Because for the outliers paper that would make you ineligible so I've chosen McGill but since I'm not a student there anymore I think I'm not eligible anyway. Should I change it to yours?

**Comment by strengejacke** [comment]
> Daniel is it true your institution will only reimburse if you are first/senior author?

Yes, that's the way the scoring system works and how APC are reimbursed.

**Comment by rempsyc** [comment]
> fix the Unicode issue like you did before?

The workaround last time was to change the font because for some reasons the default font in these LaTex formats doesn't support Hebrew. I don't know if we're allowed to change the font given we have to use their MDPI template though. Not sure if we can even overwrite it easily.

**Comment by strengejacke** [comment]
> Because for the outliers paper that would make you ineligible

Yes, possible APC for that paper must be covered in a different way :-/

**Comment by rempsyc** [comment]
I confirm that adding `mainfont: Times New Roman` does not solve the LaTeX error. Hum, what a pickle.

**Comment by bwiernik** [comment]
Which latex engine are you using? Need to use XeTeX or LuaLaTeX for Unicode support

**Comment by rempsyc** [comment]
Thanks. I tried both `xelatex` and `lualatex`, and it seems to have worked because I'm getting a new error:

```
! Undefined control sequence.
\set@color ->\pdfcolorstack 
                            \@pdfcolorstack push{\current@color }\aftergroup...
l.1462 \color{black}

Error: LaTeX failed to compile paper.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See paper.log for more info.
Execution halted
```

Unfortunately, I'm not able to to debug this error as I'm a Microsoft Word guy, I've never used LaTex before those _easystats_ papers 😬. However, what I've tried is using `lualatex` or `xelatex` engines on the outliers paper that we otherwise know for a fact that knit well, and got the same error. This leads me to believe that the current Mathematics template does not support those alternative engines.

I believe this is confirmed here: https://github.com/rstudio/rticles/issues/502

Also seems like the `rticles` MDPI template is outdated and someone has opened a pull-request with the updated version, however, that version has not been merged: https://github.com/rstudio/rticles/pull/515

What's the likelihood we're going to get help from the `rticles` maintainers? And what to do if we can't get help?

**Comment by bwiernik** [comment]
I can take a look today

**Comment by strengejacke** [comment]
> And what to do if we can't get help?

We knit to word, using mdpi template, and then edit the special chars in word

**Comment by bwiernik** [comment]
I've got a fix

**Comment by bwiernik** [comment]
@mattansb Really nice blog post we should cite https://psychometroscar.com/2022/04/21/the-relationship-between-the-phi-coefficient-and-the-chi-square-test-of-association/

**Comment by mattansb** [comment]
Feel free to add that anywhere you see fit (:

**Comment by strengejacke** [comment]
> I've got a fix

That's great! Would you mind fixing the issue here then? 😬

**Comment by bwiernik** [comment]
Eh, looks like I was wrong there. Still something broken in the MDPI TeX template or in rticles implementation. 

Let's just compile to Word and submit like heathens?

**Comment by rempsyc** [comment]
@mattansb authors should give final approval before submission. Is this the final version ready to be submitted that we can give final approval to?

**Comment by strengejacke** [comment]
> ready to be submitted

No, I think we need a Word version to get the correct special char, then we can approve that manuscript and submit.

**Comment by bwiernik** [comment]
MDPI encourages but doesn't require using their latex or word templates. Do you think we can just submit a generic TeX or word doc or do you want to bother with their template? 

I can get either format to work--it's the rticles MDPI template that's got bugs in it related to syntax highlighting colors that are the issue

**Comment by rempsyc** [comment]
I think it would be nice to use their template—assuming that's not overly complicated.

**Comment by bwiernik** [comment]
Probably the least effort full way to do that at this point would be to generate either a tech file or a word document and then copy and paste the relevant sections into their spots

**Comment by bwiernik** [comment]
@rempsyc did those "new statistics" changes make it into the Rmd?

**Comment by rempsyc** [comment]
No, as we have abandoned the Rmd to work exclusively on the word doc, in my understanding.

**Pull request merged**
